charts/alfresco-sync-service/ci/kustomize/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+---
+namespace: alfresco
+configMapGenerator:
+  - name: dummy-repo
+    files:
+      - authentication
+      - config
+  - name: httpd-conf
+    files:
+      - httpd.conf
+
+helmChartInflationGenerator:
+  - chartName: apache
+    chartRepoUrl: https://charts.bitnami.com/bitnami
+    chartVersion: 9.2.18
+    releaseName: dummy-repo
+    releaseNamespace: alfresco
+    values: values.yaml
```

Comment:
Things to consider:
* we already have [pre-requisites](https://github.com/Alfresco/alfresco-helm-charts/blob/632758d9c285321b367976393cc2fd825c65a5a4/.github/workflows/lint-test.yaml#L74) in the workflow.
* Here we're essentially trying to workaround a k8s unfriedly behavior of sync service (the real fix would be to have sync-service not fails miserably if it can't find a repo at startup).
The other options I considered:
* add that as a dependency of the chart => having that dummy stuff in a chart is really 
* adding a true repo chart dependency => we don't have such chart right now & implementing it is way beyond the scope of the ticket